pom.xml

```diff
@@ -150,6 +150,7 @@
     <!--  org.apache.httpcomponents/httpclient-->
     <commons.httpclient.version>4.5.4</commons.httpclient.version>
     <commons.httpcore.version>4.4.8</commons.httpcore.version>
+    <trove.version>3.0.1</trove.version>
```

Comment:
it looks like this dep is being taken on for a couple data structures: are these important or could they be replaced with scala Array and HashMap?

Reply:
My take is that the performance is probably not worth the additional dependency.
I also noticed that the trove dep is LGPL, which is considered incompatible with Apache license. Although I believe this is not a show-stopper with respect to "containerized" dependencies, it probably is for a direct dependency in the code.

Reply:
Noted, will remove in the next push.